modules/ROOT/pages/munit-maven-plugin.adoc

```diff
@@ -412,51 +367,55 @@ INFO] [CoverageManager] Printing Coverage Report...
 [INFO]  * Application Coverage: 71.43%
 [INFO]
 [WARNING] ----------------------------- WARNING --------------------------------------
-[WARNING]  * Application coverage is below defined limit. Required: 100.0% - Current: 71.43%  //<1>
+[WARNING]  * Application coverage is below defined limit. Required: 100.0% - Current: 71.43%
 ----
-<1> Warning detailing which coverage level wasn't met
 
-TIP: If no level is defined, -1 is assumed, which indicates that the build won't fail due to lack of coverage.
+If you don't define a coverage level, MUnit Coverage assumes a `-1` value, which indicates that the build won't fail due to lack of coverage.
 
-== To Ignore Flows for Coverage Report
+=== Ignoring Flows for Coverage Report
 
 .MUnit Coverage - Ignoring Flows
 [source,xml,linenums]
 ----
 <coverage>
   <ignoreFlows>
-	  <ignoreFlow>the-name-of-your-flow</ignoreFlow>       //<1>
+	  <ignoreFlow>the-name-of-your-flow</ignoreFlow>
   </ignoreFlows>
 </coverage>
 ----
-<1> The name of the flow you want to ignore.
 
-This is a list, so you can ignore as many flows as you need.
+`<ignoreFlows>` is a list, so you can ignore as many flows as you need:
+
+[source,xml,linenums]
+----
+<coverage>
+  <ignoreFlows>
+      <ignoreFlow>the-name-of-your-flow</ignoreFlow>
+      <ignoreFlow>the-name-of-another-flow</ignoreFlow>
+  </ignoreFlows>
+</coverage>
+----
 
 == Dynamic Ports
 
-[IMPORTANT]
-MUnit 2.2 and later introduce the `dynamic-port` global element, that allows you to define dynamic ports at the MUnit suite level. +
-Using this element instead of the plugin configuration described below allows you to set the dynamic port both from Maven and Studio. +
+MUnit 2.2 and later introduce the `dynamic-port` global element, that allows you to define dynamic ports at the MUnit suite level. Using this element instead of the plugin configuration described as follows allows you to set the dynamic port both from Maven and Studio. +
```

Comment:
This phrase is confusing: 
Using this element instead of the plugin configuration described as follows allows you to set the dynamic port both from Maven and Studio.

Isn't this section describing how to configure dynamic ports? If so, why does it say "instead of the plugin configuration described as follows" 

It's ok to replace it with just: 
Using this element enables you to set the dynamic port both from Maven and Studio. ?